style.css

```diff
@@ -5,4 +5,12 @@ body {
 
 a {
     color: #fff;
+}
+
+header {
+    background-color: salmon;
```

Comment:
I'm really not too sure about this colour, Zsolt. What were you thinking?

Reply:
Actually I think this colour is fantasitc